local/local.go

```diff
@@ -547,6 +550,11 @@ func (o *Object) Storable() bool {
 	return true
 }
 
+// Return the map of metadata
+func (o* Object) UserMetadata() map[string]string {
+	return o.metadata
```

Comment:
`o.metadata` could be nil here.  Go will happily `range` a nil map, and that might be the way to do it, but there should be a comment somewhere about that.